### PR TITLE
Mlt families

### PR DESCRIPTION
**Pull request by morfi11**

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
                 {
                   "type": "initial",
                   "maximumWarning": "2mb",
-                  "maximumError": "5mb"
+                  "maximumError": "6mb"
```
Comment (noam-honig) [inline angular.json:60]: is this still necessary?

File: src/app/delivery-history/delivery-history.component.ts [hunk 2]
```diff
@@ -315,8 +348,10 @@ export class helperHistoryInfo extends Entity<string>{
   selfassigned = new NumberColumn(getLang(this.context).selfAssigned);
   families = new NumberColumn(getLang(this.context).families);
   dates = new NumberColumn(getLang(this.context).dates);
+  giftsConsumed = new NumberColumn('מתנות שמומשו');
+  giftsPending = new NumberColumn('מתנות זמינות');
   constructor(private context: Context) {
-    super({ name: 'helperHistoryInfo', allowApiRead: false, allowApiCRUD: false });
+    super({ name: 'helperHistoryInfo', allowApiRead: true, allowApiCRUD: false });
```
Comment (noam-honig) [inline src/app/delivery-history/delivery-history.component.ts:354]: Y?

File: src/app/families/DeliveryStatus.ts [hunk 3]
```diff
@@ -29,7 +35,12 @@ export class DeliveryStatus {
   static FailedBadAddress: DeliveryStatus = new DeliveryStatus(21, !use ? '' : use.language.notDeliveredBadAddress, true);
   static FailedNotHome: DeliveryStatus = new DeliveryStatus(23, !use ? '' : use.language.notDeliveredNotHome, true);
   static FailedDoNotWant: DeliveryStatus = new DeliveryStatus(24, !use ? '' : use.language.notDeliveredDontWant, true);
-  static FailedOther: DeliveryStatus = new DeliveryStatus(25, !use ? '' : use.language.notDeliveredOther, true);
+
+  static FailedNotReady: DeliveryStatus = new DeliveryStatus(26, !use ? '' : "התרומה עוד לא מוכנה לאיסוף", true);
+  static FailedAlreadyPickedUp: DeliveryStatus = new DeliveryStatus(27, !use ? '' : "התרומה כבר נמסרה", true);
+  static FailedTooFar: DeliveryStatus = new DeliveryStatus(28, !use ? '' : "רחוק לי", true);
+
+  static FailedOther: DeliveryStatus = new DeliveryStatus(29, !use ? '' : use.language.notDeliveredOther, true);
```
Comment (noam-honig) [inline src/app/families/DeliveryStatus.ts:43]: You ignored my last comments ?

Comment (noam-honig) [reply src/app/families/DeliveryStatus.ts:43]: you can't change the id of an existing status - without affecting existing data or migrating it.

**Review by noam-honig — CHANGES_REQUESTED** [review]

File: src/app/delivery-history/delivery-history.component.ts (hunk 2 above)
Comment (morfi11) [reply src/app/delivery-history/delivery-history.component.ts:354]: probably not. but not really harmful. anyway - changed it back

File: angular.json (hunk 1 above)
Comment (morfi11) [reply angular.json:60]: yes :(

**Pull request merged**